autoload/ctrlp/OmniSharp/findsymbols.vim

```diff
@@ -60,7 +59,20 @@ endfunction
 " Return: a Vim's List
 "
 function! ctrlp#OmniSharp#findsymbols#init() abort
+<<<<<<< HEAD
   return s:symbols
+=======
+  if !OmniSharp#ServerIsRunning()
+    return
+  endif
+
+  let s:quickfixes = OmniSharp#py#eval('findSymbols()')
+  let symbols = []
+  for quickfix in s:quickfixes
+    call add(symbols, quickfix.text)
+  endfor
+  return symbols
+>>>>>>> 1eb03cb... Remove #py#exists and fixup the new util function
```

Comment:
This merge got missed.

The HEAD version is correct - this function no longer makes a python call, the symbols are passed in (via `ctrl#OmniSharp#findsymbols#setsymbols()` above), so this function just returns them:
```vim
function! ctrlp#OmniSharp#findsymbols#init() abort
  return s:symbols
endfunction
```